src/generators.ts

```diff
@@ -37,6 +37,10 @@ export const idGenerators = {
     return join(['http_operation', props.parentId, props.method, sanitizePath(props.path)]);
   },
 
+  httpCallbackOperation: (props: Context & { method: string; path: string }) => {
+    return join(['http_operation', props.parentId, props.method, props.path]);
```

Comment:
```suggestion
    return join(['http_callback', props.parentId, props.method, props.path]);
```

Reply:
good catch! it has been updated in the latest commit